src/app/pagamentoplanos/component/pagamentoplanos.component.ts

```diff
@@ -1,15 +1,130 @@
 import { Component, OnInit } from '@angular/core';
+import { ExternalService } from '../service/external.service';
+import { PagamentoplanosService } from '../service/pagamentoplanos.service';
 
 @Component({
   selector: 'app-pagamentoplanos',
   templateUrl: './pagamentoplanos.component.html',
   styleUrls: ['./pagamentoplanos.component.css']
 })
 export class PagamentoplanosComponent implements OnInit {
+  cardForm: any;
 
-  constructor() { }
+  constructor(
+    private scriptService: ExternalService,
+    private pagamentoSerice: PagamentoplanosService
+  ) { }
 
   ngOnInit(): void {
+    this.pagamentoSerice.ngOnInit();
+    this.scriptService.load('MercadoPago').then(data => {
+  }).catch(error => console.log(error));
+    //@ts-ignore
+    const mp = new MercadoPago('TEST-3765ae52-23a2-40f0-9811-7a3d6cbbe51d');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
    const mp = new MercadoPago("TEST-3765ae52-23a2-40f0-9811-7a3d6cbbe51d");
```